README.rst

```diff
@@ -1,83 +1,74 @@
-Documentation Build Instructions
+Documentation build instructions
 ################################
 
-The `website documentation`_ for Clear Linux\* OS for Intel Architecture
-should be written in :abbr:`ReStructuredText (ReST)` AKA ``.rst``, which
-makes it easy to build parsable, command-line readable, indexed, and
-search-friendly documentation and APIs with `Sphinx`_.
-
-.. _requirements:
-
-Requirements
-============
-
-To build documentation with Sphinx, ensure your system has these
-prerequisites:
-
-* `GNU make`_
-* `Python`_
-* `PIP`_
-* `Sphinx`_
-
-The instructions for installing these varies according to OS. On a basic out-
-of-the-box Ubuntu-like OS (which usually has Python installed by default),
-check your python version you might need something like:
-
-.. code-block:: console
-
-   $ sudo apt-get install python-pip
-   $ sudo pip install -U sphinx sphinx-autobuild
+.. todo add comment re not using standards here.
 
+`Clear Linux\* OS documentation`_ is written using `reStructuredText`_ and
+built using `Sphinx`_. Documentation may be built locally for development and
+testing purposes by following the instructions in this README.
 
+Please make yourself familiar with our `contribution guidelines`_ before
+submitting a contribution.
 
-.. code-block:: console
+Requirements
+************
 
-   $ python -c 'print __import__("sphinx").__version__'
-     1.3.1
+Make sure you have Python and Sphinx installed. We use Python 3 and
+Sphinx 1.7.5
 
-Cloning the documentation repository
-====================================
+The Sphinx documentation provides `instructions for installing Sphinx`_ on various
+platforms.
 
-We have confirmed Sphinx installed.  The next step is to clone Gitlab
-repository to our local machine.
+Clone the documentation repository
+**********************************
 
+Once Sphinx is installed, clone the documentation repository to your
+local machine.
 
 .. code-block:: console
 
    $ git clone https://github.com/clearlinux/clear-linux-documentation
 
-Run make
-========
+Run the build
+*************
 
-Finally are we ready to run :command:`make`. Be sure to :command:`cd` to the
-:file:`source/` directory where your ``.rst`` files are, before
-running :command:`make html`, or the doc format of your choice.
+We build our documentation using . In the source directory of your
```

Comment:
Change: We build our documentation using Sphinx.

Reply:
Done